example/flaskr/static/js/ratingTable.js

```diff
@@ -0,0 +1,310 @@
+    //             nextTarget = targets[1];
+    //             nextTarget.className = "alts selected";
+    //         }else{
+    //             nextTarget = targets[i + 1];
+    //             nextTarget.className = "alts selected";
+    //         }
+    //         return;
+    //     }
+    // }
+}
+
+function changeAmount(currentTarget, signal){
+    // Depending on signal, either increase or decrease amount by .1
+    target = currentTarget.querySelector(".amount");
+    inputs = target.innerHTML.split(" ");
+    returnInput = "";
+    if(signal === "increase"){
+        inputs[0] = Math.round((parseFloat(inputs[0]) + .1) * 100)/100;
+        for(i in inputs){
+            returnInput += inputs[i] + " ";
+        }
+        target.abbr = inputs[0] + " " + inputs[1];
+        target.innerHTML = returnInput;
+        console.log(target.abbr);
+    }
+    else if(signal === "decrease"){
+        inputs[0] = Math.round((parseFloat(inputs[0]) - .1) * 100)/100;
+        for(i in inputs){
+            returnInput += inputs[i] + " ";
+        }
+        target.abbr = inputs[0] + " " + inputs[1];
+        target.innerHTML = returnInput;
+    }else{return;}
+}
+
+function eventManager(event){
+    // handles events from ingredient table and call corresponding function
+    target = event.target;
+    curtarget = event.currentTarget;
+    classes = target.classList;
+    if(target.classList.contains("alts")){
+        changeAlts(target);
+    }
+    else if(target.classList.contains("amount")){
+        if(classes.contains("increase")){
+            changeAmount(curtarget, "increase");
+        }else if(classes.contains("decrease")){
+            changeAmount(curtarget, "decrease");
+        }
+    }
+    else{return;}
+}
+
+function updateRecipe(event){
+    //console.log(event.currentTarget);
+}
+// add event listener to ingredient table
+Array.from(document.getElementsByClassName('alts')).forEach(selected => {
+    selected.onchange = function(){
+        var id = selected.value;
+        var name, category;
+        for(name in alts){
+            if(parseInt(alts[name]["id"]) == selected.value){
+                category = alts[name]["category_id"];
+                break;
+            }
+        }
+        for(item in ingredients){
+            if(ingredients[item]["category"] == category){
+                amount = ingredients[item]["amount"];
+                unit = ingredients[item]["unit"];
+                recipe_id = ingredients[item]["unit"];
+                delete ingredients[item];
+
+                ingredients[name] = {};
+                ingredients[name]["recipe_id"] = recipe_id;
+                ingredients[name]["ingredient_id"] = selected.value;
+                ingredients[name]["category"] = category;
+                ingredients[name]["amount"] = amount;
+                ingredients[name]["unit"] = unit;
+                calcSubRatings(ingredients, alts);
+                calcCategoryRatings();
```

Comment:
Need to add statements here to also change the colored ratings in the 'Pros' and 'Cons' list.

Reply:
Thanks, that will be to next PR update! :)
Will merge for now and continue working on the pros/cons column! :+1: